examples/jetstream/api-migration/deno/main.ts

```diff
@@ -0,0 +1,207 @@
+import {
+  AckPolicy,
+  connect,
+  consumerOpts,
+} from "https://deno.land/x/nats@v1.14.0/src/mod.ts";
+
+// Get the passed NATS_URL or fallback to the default. This can be
+// a comma-separated string. If not defined, it will default to localhost:4222
+const servers = Deno.env.get("NATS_URL")?.split(",");
+console.log(servers);
+
+// Create a client connection to an available NATS server.
+const nc = await (connect({ servers }));
+
+// Create a JetStream Manager context - this context has API we can use to
+// create streams and consumers.
+const jsm = await nc.jetstreamManager();
+// and create a stream
+await jsm.streams.add({
+  name: "EVENTS",
+  subjects: ["events.>"],
+});
+
+// We also create a JetStream context that provides a mechanism
+// for publishing data to JetStream and to create "subscriptions"
+// on a consumer:
+const js = nc.jetstream();
+// Here we add 20 messages to a stream
+const proms = Array.from({ length: 20 })
+  .map((_v, idx) => {
+    return js.publish(`events.${idx}`);
+  });
+await Promise.all(proms);
+
+// Processing messages using Legacy JetStream is fairly cumbersome
```

Comment:
Unfortunately those side notes will not be rendered correctly in NBE, so they should reside above some code for which they make sense. Right now if you build the example and run `nbe serve`, the way this is rendered looks like this ("Processing messages..." sits right next to the `subscribe` example code):


<img width="1019" alt="image" src="https://github.com/ConnectEverything/nats-by-example/assets/15184508/d7d8b347-4963-4e52-a879-7179cd6b9336">